src/main/java/uk/gov/hmcts/darts/transcriptions/validator/WorkflowValidator.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.darts.transcriptions.validator;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum;
+import uk.gov.hmcts.darts.transcriptions.enums.TranscriptionTypeEnum;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.APPROVED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.AWAITING_AUTHORISATION;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.CLOSED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.COMPLETE;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.REJECTED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.REQUESTED;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionStatusEnum.WITH_TRANSCRIBER;
+import static uk.gov.hmcts.darts.transcriptions.enums.TranscriptionTypeEnum.OTHER;
+
+@Component
+@Slf4j
+public class WorkflowValidator {
+
+    // Create a map defining the transition rules, from any given current state to a list of allowable target states
```

Comment:
I don't think we need this comment personally as it seems obvious enough from your validateChangeToWorkflowStatus variable names...

Reply:
gone